src/types/notification.ts

```diff
@@ -64,3 +65,12 @@ export const toFiscalCodeHash = (fiscalCode: FiscalCode): FiscalCodeHash => {
 
   return hash.digest("hex") as FiscalCodeHash;
 };
+
+/**
+ * Common response message type.
+ */
+export const SuccessResponse = t.interface({
+  message: string
```

Comment:
usually it's better to type t.string (and avoid to `import { string } from "io-ts";`) as it may be a source of confusion (with typescript primitive types like 'string')